### PR TITLE
Add Count Set Bits in JavaScript

### PR DESCRIPTION
**Pull request by unixzii**

### CODE REVIEW

File: code/bit-manipulation/count_set_bits/countSetBits.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+export default function countSetBits(n) {
```
Comment (arnavb) [inline code/bit-manipulation/count_set_bits/countSetBits.js]: Add OpenGenus comment.

File: code/randomized-algorithms/shuffle_an_array/shuffleArray.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
+// This is a JavaScript implementation of Fisher-Yates shuffle based on
```
Comment (arnavb) [inline code/randomized-algorithms/shuffle_an_array/shuffleArray.js]: Add OpenGenus comment.

**Comment by unixzii** [comment]
@arnavb 
Comments added.

**Comment by AdiChat** [comment]
Looks great 😍 

Hope you are enjoying your journey 🚆  with **OpenGenus**

**Pull request merged**